src/components/Articles.jsx

```diff
@@ -5,22 +5,60 @@ import { Link } from "react-router-dom";
 const Articles = () => {
     const [articles, setArticles] = useState(null);
     const [isLoading, setIsLoading] = useState(true);
-
-
-    useEffect(() => {
+    const [form, setForm] = useState({})
+  
+    
+      useEffect(() => {
       setIsLoading(true);
       fetchArticles().then((response) => { 
         setArticles(response);
          setIsLoading(false);
       });
     }, []);
+
+   
+    const handleChange = (event) => {
+      setForm((currentForm) => {
+          const information = { ...currentForm, [event.target.id]: event.target.value }
+          return information
+      })
+    }
+
+    const handleSubmit = (event) => {
+      event.preventDefault()
+      setIsLoading(true);
+      fetchArticles(form.sort_by, form.order).then((response) => { 
+        setArticles(response);
+         setIsLoading(false);
+         
+      });
+    }
     
     if (isLoading) return <h2 className= "loading">Loading...</h2>
     
     return (
       <>
-        <h2 className = "articlesTitle">Latest News</h2>
-        
+        <h2 className = "articlesTitle">All Articles</h2>
+  
+        <div className="filterArticles">
+            <form action="">
+         <label htmlFor="sort_by" className="sortBy">Sort By:</label>
+         <select id="sort_by" className="filterSortButton" onChange={handleChange}  >
+           <option  value="created_at">Date</option>
+           <option  value="article_id">Article ID</option>
+           <option  value="votes">Votes</option>
```

Comment:
I suggest you include each of the values that a user can sort by on your article cards. For example, at the moment, if i sort by votes, I might see the cards change order, but there's no visual feedback on the card that the highest voted for card is at the top.